packages/typography/src/Body/Body.tsx

```diff
@@ -28,14 +28,16 @@ const fontWeights: Record<
 } as const;
 
 export function Body<T extends keyof JSX.IntrinsicElements>({
+  baseFontSize: baseFontSizeOverride,
   darkMode: darkModeProp,
   className,
   weight = 'regular',
   as = 'p' as T,
   ...rest
 }: BodyProps<T>) {
   const { theme } = useDarkMode(darkModeProp);
-  const baseFontSize = useUpdatedBaseFontSize();
+  const providerBaseFontSize = useUpdatedBaseFontSize();
+  const baseFontSize = baseFontSizeOverride ?? providerBaseFontSize;
```

Comment:
I believe you can pass the override to `useUpdatedBaseFontSize(baseFontSizeOverride)`

Reply:
Updated - re r?